packages/gatsby-theme-newrelic/src/components/GlobalHeader.js

```diff
@@ -418,12 +426,20 @@ const GlobalHeader = ({ className, activeSite }) => {
             >
               <Button
                 as={ExternalLink}
-                size={Button.SIZE.EXTRA_SMALL}
+                size={Button.SIZE.SMALL}
                 variant={Button.VARIANT.LINK}
                 href="https://one.newrelic.com"
                 css={css`
+                  margin: 0 10px;
```

Comment:
```suggestion
                  margin: 0 0.625rem;
```

We like to use `rem` for margins. Let's update to use `rem` here.